vue/components/ui/organisms/details-expandable/details-expandable.vue

```diff
@@ -0,0 +1,178 @@
+                />
+            </div>
+            <div
+                class="details-expandable-row"
+                v-for="(value, name, subIndex) in section"
+                v-bind:key="subIndex"
+            >
+                <slot v-bind:name="sectionName + '-label-' + name">
+                    <label-ripe
+                        class="details-expandable-title"
+                        v-bind:text="capitalizeName(name)"
+                        v-bind:font-size="labelFontSize"
+                    />
+                </slot>
+                <div class="details-expandable-value">
+                    <slot v-bind:name="sectionName + '-' + name" v-bind:field-value="value">
+                        <input-ripe v-bind:value="value" />
+                    </slot>
+                </div>
+            </div>
+        </div>
+        <div class="details-expandable-btn-container">
+            <button-color
+                class="details-expandable-btn-discard"
+                v-bind:size="'small'"
+                v-bind:text="discardButtonText"
+                v-bind:secondary="true"
+                v-bind:icon="'close'"
+                v-on:click="onDiscardHandler"
+            />
+            <button-color
+                class="details-expandable-btn-save"
+                v-bind:size="'small'"
+                v-bind:text="saveButtonText"
+                v-bind:icon="'save'"
+                v-on:click="onSaveHandler"
+            />
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+.details-expandable {
+    padding: 0px 16px 0px 16px;
+}
+
+.details-expandable-caret {
+    height: 24px;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 24px;
+}
+
+.details-expandable-btn-container {
+    float: right;
+    height: 25px;
+    margin-top: 50px;
+}
+
+.details-expandable-btn-discard {
+    margin-right: 6px;
+    min-width: 125px;
+}
+
+.button-color.button-color-small {
+    min-width: 125px;
+}
+
+.details-expandable-section {
+    cursor: pointer;
+    display: flex;
+}
+
+.details-expandable-section-hide {
+    border-bottom: 1px solid #e4e8f0;
+    height: 25px;
+    margin-top: 25px;
+    overflow: hidden;
+    padding-bottom: 25px;
+    position: relative;
+}
+
+.details-expandable-section-show {
+    height: auto;
+    margin-top: 25px;
+    overflow: visible;
+    padding-bottom: 25px;
+    position: relative;
+}
+
+.details-expandable-header {
+    color: #0d0d0d;
+    font-size: 12px;
+    font-weight: 700;
+    letter-spacing: 0.6px;
+    text-transform: uppercase;
+}
+
+.details-expandable-title {
+    color: #57626e;
+}
+
+.details-expandable-value {
+    margin-top: 10px;
+}
+
+.details-expandable-row {
+    margin-top: 25px;
+}
+</style>
+<script>
+export const DetailsExpandable = {
+    name: "details-expandable-ripe",
+    props: {
+        data: {
+            type: Object,
+            required: true
+        },

```

Comment:
This type of prop is problematic and we should avoid it as it's too general. Usually we use Arrays (which isn't necessarily a lot more "specific") with a defined template on how to define key -> value pairs.

Also `data` is a very general name for a prop, if you can use a more informative name I think it would be best. Something like `fields` or something similar. Check and follow the standard in other components.